traefik/Chart.yaml

```
@@ -3,7 +3,7 @@ name: traefik
 description: A Traefik based Kubernetes ingress controller
 type: application
 version: 10.24.2
```

The `version` will need incrementing here, `appVersion` shouldn't be touched as it's related to the Traefik version and not the Helm chart version
```suggestion
version: 10.25.0
```